### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by Kosterio** [open-webrtc-toolkit/owt-client-native#712]
I need a QUIC support on android client side, and apparently it impossible now.
It seems that the owt-client-android is based on pure webrtc library without owt, isn't it?
Are there plans to add support to the native library with owt for the Android app?

**Comment by jianjunz** [comment]
Hi,

There is no plan to support QUIC for Android at this time. QUIC is implemented in [owt-sdk-quic](https://github.com/open-webrtc-toolkit/owt-sdk-quic), OWT Android SDK doesn't depend on that repo.

**Comment by Kosterio** [comment]
Yes, QUIC is implemented in another repo, but Android cannot use it - this is the problem.
Do I understand correctly that only the JNI layer implementation is missing for the native library to work on Android?

**Comment by jianjunz** [comment]
I haven't tried to compile it for Android. Not sure if there is anything else missing.
